Features4/tests/feature/addContact.feature

```diff
@@ -0,0 +1,23 @@
+Feature:
+    As user
+    I want to add contacts
+    So that I can save my contacts
+
+Background:
+	Given the user has logged in with the valid email "deekshyap@gmail.com" and password "123456"
+    And the user navigated to the homepage 
+
+Scenario: Add multiple contacts 
+    When the user adds with the following credentails
+        | Name | Email           | Phone       | Contact Type |
+        | Hari | hari@gmail.com  | 9477859959  | Personal     |
+        | Xyz  | xyap@gmail.com  | 8549560060  | Professional |
+    Then the contacts with name as "<Name>", email as "<email>",phone as "<Phone>" and  Contact type as "<Contact Type>"should be displayed
+
+Scenario Outline: User added invalid email format
+     When the user added name "<Name>" ,invalid email "<Email>" ,phone "<Phone>" and Contact Type "<Contact Type>"
```

Comment:
This action is the same as in Scenario: Add multiple contacts - When step.
See if we can reuse the same step.